posts/atomic-design-bootstrap.md

```diff
@@ -0,0 +1,154 @@
+
+# Ok, but, what should I use if I want to use Atomic Design?
+
+Like I said, you must avoid _Bootstrap_ components usage because they may come into conflict with __Functional Patterns__ and __Perceptual Patterns__  of your design.
+
+However, _Bootstrap_ has some useful elements with less impacts on __Functional Patterns__ and  __Perceptual Patterns__.
+
+Let's start with some examples.
+
+## Grid system
+
+![A Bootstrap grid system disposition](https://raw.githubusercontent.com/gnuk/blog-usa/atomic-bootstrap/images/2020/01/atomic-design-bootstrap-grid.png)
+_Bootstrap grid system example from the website_
+
+One of these elements is the [grid system](https://getbootstrap.com/docs/4.4/layout/grid), because it allow a grid simulation with 12 columns an manage some responsive design issues. Even if gutters (space between columns: 30px by default) may have a lot of effects on the building of a grid, they can be removed using `.no-gutters` modifier.
+
+Related to `Components`, the gris has a less impact on a perceptive and functional value. It is used to structure elements inside columns and can be used on the `organism` part to manage `molecules` and `atoms`.
+
+## Utilities
+
+Some elements are use as utilities, they usually have a lower graphical value than `Components` and are used by composition..
+
+As an example, we will use [text](https://getbootstrap.com/docs/4.4/utilities/text).
+
+This utility let you do generics actions like:
+
+* alignment
+* prevent line return and truncate
+* transform to lowercase, uppercase …
+* change weight and use italic
+* change font style to monospace
+
+Some rules are more generic than others: where alignment is common and will have a small effect on style, the specific font choice for something like monospace may have more impacts:
+
+```css
+.text-monospace {
+    font-family: SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace;
+}
+```
+
+May be you can think it is not an important effect, but, depending on the utilities, this type of effect may vary, this is the point of [colors](https://getbootstrap.com/docs/4.4/utilities/colors) which has its own semantic and refers to _Bootstrap_ branding.
+
+You must be pragmatic to choose utilities.
+
+# Go further with Sass
+
+Extends and customize _Bootstrap_ can be done using  [Sass (SCSS)](https://github.com/twbs/bootstrap) [sources](https://sass-lang.com). It is possible to change container size depending on window size, colors, and there is many other exposed variables using `!default` Sass principle. Moreover, file fragmentation let you use parts of _Bootstrap_ like use only the grid system for example by importing the Sass file you want to use.
```

Comment:
Extending and customising*

and there are many other*

file fragmentation lets*

like using only the grid system